webapp/src/pages/RegisterPage/RegisterPage.css

```diff
@@ -0,0 +1,72 @@
+.register {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  min-height: 40rem;
+  overflow: scroll;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  grid-template-columns: repeat(var(--column-number), 1fr);
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    'title title title title'
+    'main main main main';
+}
+
+.register-title > *,
+.register-subtitle > * {
+  margin: 0;
+  margin-bottom: 4rem;
+}
+
+.register-title {
+  grid-area: title;
+}
+
+.register-container {
+  grid-area: main;
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem var(--column-gap);
+}
+
+.register-form .inputs-container > .input {
+  margin-bottom: 0.875rem;
+  width: 100%;
+  min-width: 15rem;
+  max-width: 20rem;
+}
+
+.register-form .inputs-container > p:empty::before {
+  content: '';
+  display: inline-block;
+}
+
+.register .landing-animation {
+  display: none;
+}
+
+/* Styles for viewports wider than 768 pixels. */
+@media (min-width: 768px) {
+  .register {
+    min-height: 20rem;
+    padding: 4rem var(--padding-sides) 8rem var(--padding-sides);
+    grid-template-columns: repeat(var(--column-number), 1fr);
+    grid-template-rows: repeat(2, auto);
+    grid-template-areas:
+      'title title title title title title ... ... ... ... ... ...'
+      'main main main main main ... ... animation animation animation animation animation';
+  }
+  .register-container {
+    grid-area: main;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem var(--column-gap);
+  }
+
+  .register .landing-animation {
+    display: inline;
+  }
+}
```

Comment:
I have a problem, again, with the height of the page. I work on a small screen, and have to do quite some scrolling even in a page with not so much content as this one, but maybe there's no clean way around my scrolling...
Where even is the figma for this particular page? Did you wing it?

Reply:
Yes, the ticket is open, once I fix the other one I will fix this as well. BTW I couldn't do what you suggested, `scroll: clip `, what did you mean?

Reply:
I'm not sure. It was pseudo code, not really code. I will try to find a solution for this tomorrow.